### PR TITLE
PS-9307 Release notes for Percona Server for MySQL 8.0.38/8.0.39-30

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/8.0.38.md
	new file:   docs/release-notes/8.0.39-30.md
	modified:   docs/release-notes/release-notes_index.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0.38.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Percona Server for MySQL 8.0.38
+
+Due to a critical issue that could prevent the server from restarting after creating a large number of tables (8001 or more), MySQL has removed the 8.0.38 release. MySQL has released 8.0.39 shortly (22 days later) after MySQL 8.0.38. Percona has skipped the release of Percona Server for MySQL 8.0.38. The next release of Percona Server for MySQL is Percona Server for MySQL 8.0.39-30 which contains all bug fixes and contents of both MySQL 8.0.38 and MySQL 8.0.39.
```
Comment (patrickbirch) [inline docs/release-notes/8.0.38.md]: Due to a critical fix, MySQL Community Server 8.0.39 was released shortly (22 days later) after MySQL Community Server 8.0.38. Percona has skipped the release of Percona Server for MySQL 8.0.38. Percona Server for MySQL 8.0.39 contains all bug fixes and contents from MySQL Community Server 8.0.38 and MySQL Community Server 8.0.39.

File: docs/release-notes/8.0.38.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## Useful links
```
Comment (patrickbirch) [inline docs/release-notes/8.0.38.md]: We don't need the Useful Links on this RN.

File: docs/release-notes/8.0.38.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.38.md]: updated

File: docs/release-notes/8.0.38.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.38.md]: updated

File: docs/release-notes/8.0.39-30.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+* [PS-9314](https://perconadev.atlassian.net/browse/PS-9314): There was no connection to the server because mysqld received signal 11 when using `JSON_TABLE`.
+
+* [PS-9286](https://perconadev.atlassian.net/browse/PS-9286): The Key Management Interoperability Protocol (KMIP) component left keys in a `pre-active` state.
+
+* [PS-9306](https://perconadev.atlassian.net/browse/PS-9306): MySQL versions 8.0.38, 8.4.1, and 9.0.0 exited upon restarting if the database contained 10,000 or more tables.
+
+* [PS-9322](https://perconadev.atlassian.net/browse/PS-9322): When `super_read_only=1` is enabled, undo truncation couldn't update the data dictionary (DD), resulting in orphaned truncate log files.
+
+
```
Comment (percona-ysorokin) [inline docs/release-notes/8.0.39-30.md:42]: `PS-9379` and `PS-9144` are missing

**Review by percona-ysorokin — CHANGES_REQUESTED** [review]

File: docs/release-notes/8.0.39-30.md (hunk 3 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0.39-30.md:42]: Added.

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Review by patrickbirch — APPROVED** [review]
good

**Pull request merged**